popUp.css

```diff
@@ -0,0 +1,195 @@
+#overlay {
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  background: #ebf0ee;
+  width: 100vw;
+  height: 108vh;
+  opacity: 0.9;
+  z-index: 100;
+  display: none;
+}
+
+.popUp {
+  width: 90vw;
+  margin-left: 20px;
+  padding: 8px 0 0 0;
```

Comment:
[ ] Kindly make the under background blurry when the popup windows is open, this will make the popup matches the Figma more closely 👍🏿 you can reduce your `opacity` value


- [ ]  Please make your popup window to be closed to the figma design some are not matching the Figma design yet: 

Your Implementation | Figma Design
:---:|:---:
![xxxxpppp](https://user-images.githubusercontent.com/79058364/162008681-7a6f0ef3-e7ae-44bb-8e32-db24842a1edb.png)| ![goodpopmobile](https://user-images.githubusercontent.com/79058364/161747369-ec03ea99-2765-44b2-801a-102ed003f51a.png)





Your Implementation | Figma Design
:---:|:---:
![pppppppppp](https://user-images.githubusercontent.com/79058364/162010740-fb559ae4-0066-4268-a52f-ae7b1e3daeca.png)| ![dessssssssssssssssssssssssssss](https://user-images.githubusercontent.com/79058364/161750695-065dbf52-9b37-4ace-b217-510505df9617.png)

Reply:
Thank you so much for the observations, we will work on it

Reply:
Thank you so much for the observations, we will work on it